### PR TITLE
opt: using google/cloud-storage.

### PR DESCRIPTION
**Pull request by sinkcup**
#74

### CODE REVIEW

**Comment by zoidbergwill** [comment]
[0.21](https://packagist.org/packages/google/cloud#v0.21.0) of google/cloud goes back to the 22nd of February 2017, while [1.3.0](https://packagist.org/packages/google/cloud-storage#v1.3.0) of google/cloud-storage only goes back to the 15th of December 2017. Even [1.0.0](https://packagist.org/packages/google/cloud-storage#v1.0.0) only goes back to March 2017. 

We should maybe try support from 1.0.0 onwards, and test each latest minor version at least, and there aren't too many so we can just add them to the env matrix .travis.yml?

**Comment by sinkcup** [comment]
@zoidbergwill good idea! I have changed it.

File: composer.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
 	"require": {
 		"php": ">=5.5.0",
 		"league/flysystem": "~1.0",
-		"google/cloud": ">=0.21 <0.50"
+		"google/cloud-storage": "^1.0"
```
Comment (zoidbergwill) [inline composer.json]: I think we should do`>= 1.0 <1.4`, because it is safer.

**Comment by sinkcup** [comment]
fine, I have changed it again.

**Pull request merged**